massa-protocol-exports/src/protocol_controller.rs

```diff
@@ -157,6 +157,20 @@ impl ProtocolCommandSender {
             })
     }
 
+    /// Propagate a batch of operation ids (from pool).
+    ///
+    /// note: Full `OperationId` is replaced by a `OperationPrefixId` later by the worker.
+    pub fn propagate_operations_sync(&mut self, operations: Storage) -> Result<(), ProtocolError> {
+        massa_trace!("protocol.command_sender.propagate_operations", {
+            "operations": operations.get_op_refs()
+        });
+        self.0
+            .blocking_send(ProtocolCommand::PropagateOperations(operations))
```

Comment:
I think try_send here could be better

Reply:
propagate_endorsements use blocking_send as well as the tokio mpsc is bounded. Why do you think 'try_send' is better and what do you think should be done if try_send fails?

Reply:
Because I don't think you need to block here you are not waiting any data and I don't know why propagate_endorsement is blocking. It could be non-blocking also

Reply:
I'm not against using try_send here but I still don't know what to do if the try_send fails when the queue is full. With blocking_send, we ensure that we will propagate ops...

Reply:
Yes but we wait a lot. I think that propagating denunciations isn't a mandatory thing and so the network can run without it so if we can't send because the channel is full we just don't do anything IMO.